WooCommerce/Classes/Extensions/String+Helpers.swift

```diff
@@ -63,4 +63,9 @@ extension String {
 
         return newText
     }
+
+    /// A Boolean value indicating whether a string has characters.
+    public var isNotEmpty: Bool {
```

Comment:
Same concern with the Array helper above. 

```suggestion
var isNotEmpty: Bool {
```